app/views/spree/products/show.html.erb

```diff
@@ -1,35 +1,18 @@
 <% @body_id = 'product-details' %>
 
 <% cache [I18n.locale, current_pricing_options, @product] do %>
-  <article itemscope itemtype="http://schema.org/Product">
-    <section>
-      <%= render partial: 'image' %>
-    </section>
-    <section>
-      <%= render partial: 'thumbnails' %>
-    </section>
-    <section>
-      <%= render partial: 'properties' %>
-    </section>
-    <section>
-      <%= render partial: 'promotions' %>
-    </section>
+  <article class="product-page" itemscope itemtype="http://schema.org/Product">
+    <div class="product-page__images">
+      <%= render partial: 'spree/components/products/product_image' %>
+      <%= render partial: 'spree/components/products/product_thumbnails' %>
+    </div>
 
-    <header>
-      <h1 class="product-title" itemprop="name">
-        <%= @product.name %>
-      </h1>
-      <p itemprop="description" data-hook="description">
-        <%= product_description(@product) rescue t('spree.product_has_no_description') %>
-      </p>
-    </header>
-
-    <section>
-      <%= render partial: 'cart_form' %>
-    </section>
-
-    <footer>
-      <%= render partial: 'taxons' %>
-    </footer>
+    <div class="product-page__info">
+      <%= render partial: 'spree/components/products/product_info' %>
+      <%= render partial: 'spree/components/products/product_properties' %>
+      <%= render partial: 'spree/components/products/product_promotions' %>
+      <%= render partial: 'spree/components/products/product_form' %>
+      <%= render partial: 'spree/components/products/product_taxons' %>
```

Comment:
We can remove `partial: ` here as well. 

Also, I'd name this files without the `product_` part, since they already are into the `components/products/` folder, and that piece is not really adding anything. Or am I missing something?

Reply:
The reasoning is that it is way easier to find (in the filesystem and in the code) and reason about, since the aim is to make the template file, scss file and class name all share the same name. What do you think?

Reply:
So the concept is that the file `product_taxons` contains a component with class `product-taxons`, right? If yes, I think it's a good reason to keep it as you proposed.

Reply:
Correct! Way easier to search.